packages/core/src/lib/light_push/status_codes.ts

```diff
@@ -0,0 +1,71 @@
+import { ProtocolError } from "@waku/interfaces";
+
+export enum LightPushStatusCode {
+  SUCCESS = 200,
+  BAD_REQUEST = 400,
+  UNSUPPORTED_PUBSUB_TOPIC = 404,
+  REQUEST_TOO_LARGE = 413,
+  TOO_MANY_REQUESTS = 429,
+  INTERNAL_SERVER_ERROR = 500,
+  NO_PEERS_TO_RELAY = 503
+}
+
+export function lightPushStatusCodeToProtocolError(
+  statusCode: number
+): ProtocolError | null {
+  switch (statusCode) {
+    case LightPushStatusCode.SUCCESS:
+      return null;
+
+    case LightPushStatusCode.BAD_REQUEST:
```

Comment:
it seems we are loosing whole purpose of moving to `lightPush v3` 😆 

please, add new properties to `ProtocolError` to reflect new error cases
(later we should remove ProtocolError all together and introduce LightPushError)